eng/pipelines/jobs/fuzz.yml

```diff
@@ -18,46 +20,56 @@ parameters:
   # doesn't stop when it should.
   fuzzTimeoutMinutes: 200
   jobTimeoutMinutes: 220
+  # The list of runner IDs: sequential numbers starting from 0. The number of elements determines
+  # the number of runner jobs to start in parallel. We would use "range" here if it existed.
+  runnerIDs: [1, 2, 3]
 
 jobs:
-  - job: ${{ parameters.name }}
-    pool: ${{ parameters.pool }}
-    timeoutInMinutes: ${{ parameters.jobTimeoutMinutes }}
-    workspace:
-      clean: all
-    steps:
-      - checkout: self
-        submodules: true
-        fetchDepth: 1
-
-      - ${{ if eq(parameters.useMicrosoftGo, true) }}:
-        - template: ../steps/init-microsoft-go.yml
-          parameters:
-            platform: ${{ parameters.platform }}
-
-        - ${{ if contains(parameters.platform, 'windows') }}:
+  - ${{ each id in parameters.runnerIDs }}:
+    - job: ${{ parameters.name }}_${{ id }}
+      displayName: ${{ parameters.name }} ${{ id }}
+      pool: ${{ parameters.pool }}
+      timeoutInMinutes: ${{ parameters.jobTimeoutMinutes }}
+
+      workspace:
+        clean: all
+
+      variables:
+        bucketArgs: -bucket ${{ id }}/${{ length(parameters.runnerIDs) }}
+        ${{ if contains(parameters.platform, 'windows') }}:
+          cdFuzzCrypto: 'cd cmd\fuzzcrypto'
+          setExperiment: 'set GOEXPERIMENT='
+        ${{ else }}:
+          cdFuzzCrypto: 'set -x ; cd cmd/fuzzcrypto'
+          setExperiment: 'export GOEXPERIMENT='
+
+      steps:
+        - checkout: self
+          submodules: true
+          fetchDepth: 1
+
+        - ${{ if eq(parameters.useMicrosoftGo, true) }}:
+          - template: ../steps/init-microsoft-go.yml
+            parameters:
+              platform: ${{ parameters.platform }}
+
           - script: |
-              cd cmd\fuzzcrypto
-              set GOEXPERIMENT=cngcrypto
-              go run . -v -fuzztime ${{ parameters.fuzztime }}
+              $(cdFuzzCrypto)
+              go env
+              $(setExperiment)${{ parameters.goExperiment }}
```

Comment:
Could we use AzDO `env` instad of setting the experiment via bash or cmd notation?
Lik this:

```yaml
- script: |
    go env
    go run ...
  displayName: Fuzz
  env: ${{ parameters.goExperiment }}

```

Reply:
True. Also `workingDirectory` to get rid of the other variable.

Reply:
Ah right, `env` entries don't show up in the build logs. 😬 I usually stick with scripts with `set -x` for reproducibility/comprehensibility. (Although only on Linux.)

In this case `env` is perfectly fine because `go env` prints out the value anyway.